src/NHibernate.DomainModel/NHSpecific/NullableInt32Converter.cs

```diff
@@ -62,12 +62,12 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
 			{
 				NullableInt32 nullable = (NullableInt32) value;
 
-				System.Type[] constructorArgTypes = new System.Type[1] {typeof(Int32)};
+				System.Type[] constructorArgTypes = new[]{typeof(Int32)};
 				ConstructorInfo constructor = typeof(NullableInt32).GetConstructor(constructorArgTypes);
 
 				if (constructor != null)
 				{
-					object[] constructorArgValues = new object[1] {nullable.Value};
+					object[] constructorArgValues = new[]{nullable.Value};
```

Comment:
Hi @pawan-deepsource, autofix produced invalid result dropping the array type.

Reply:
Yup, saw it in our logs. Tracking it internally.